components/director/internal/domain/tenant/fetcher_on_demand.go

```diff
@@ -50,6 +52,10 @@ func NewFetchOnDemandService(client Client, config FetchOnDemandAPIConfig) Fetch
 
 // FetchOnDemand calls an API which fetches details for the given tenant from an external tenancy service, stores the tenant in the Compass DB and returns 200 OK if the tenant was successfully created.
 func (s *fetchOnDemandService) FetchOnDemand(ctx context.Context, tenant, parentTenant string) error {
+	if strings.Contains(tenant, "/") {
```

Comment:
Edit the comment so that the case for tenants with slashes is included (right now from the comment is expected that the code will make the request and return 200 in that case.